src/app/components/search-page/search-page.component.scss

```diff
@@ -0,0 +1,21 @@
+.label-colors {
```

Comment:
is this being used?

Reply:
Not anymore . Had a Label on top of the search bar called "Search" will remove it quick.